dom.bs

```diff
@@ -733,6 +736,12 @@ The <dfn attribute for=Event><code>bubbles</code></dfn> and
 <dfn attribute for=Event><code>cancelable</code></dfn> attributes
 must return the values they were initialized to.
 
+The <dfn attribute for=Event><code>returnValue</code></dfn>
+attribute must be initialized to true when an <a>event</a> is created.
+When the attribute is set to false it must set the <a>canceled flag</a> if the
+{{Event/cancelable}} attribute value is true and the
+<a>in passive listener flag</a> is unset.
```

Comment:
Do implementations update returnValue if the event gets canceled by e.g. preventDefault()?

Reply:
(this notification got swallowed by my inbox, will look into this tomorrow)

Reply:
@domenic yeah, Safari, Chrome and Edge do:

https://miketaylr.com/bzla/canceled-returnValue.html

So we need to add a getter that returns if the canceled flag is set, and false otherwise.